src/layout/Header/NavMenuDrop/style.scss

```
@@ -0,0 +1,67 @@
+@import "../../../style/style.scss";
+
+.nav-menu-drop {
+  @include flex($justify: space-between);
+  z-index: $z-sticky;
+  background-color: $white;
+  width: 100%;
+  position: absolute;
+  left: 0;
+  padding: 0 3rem;
+  display: none;
+  box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  @include respond-to(tablet-and-down) {
+    padding: 1rem 0;
+  }
+  .item-drop-down {
+    width: 100%;
+    .title {
+      margin-bottom: 1rem;
+      font-weight: $fw-medium;
+      @include respond-to(tablet-and-down) {
```

```suggestion
  box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
  
  @include respond-to(tablet-and-down) {
    padding: 1rem 0;
  }
  
  .item-drop-down {
    width: 100%;
    
    .title {
      margin-bottom: 1rem;
      font-weight: $fw-medium;
      
      @include respond-to(tablet-and-down) {
```
Mỗi block cách nhau 1 dòng nhé